Sming/Services/ArduinoJson/src/JsonBuffer.cpp

```diff
@@ -14,7 +14,7 @@
 using namespace ArduinoJson;
 using namespace ArduinoJson::Internals;
 
-JsonStringStorage JsonStringStorage::_invalid(NULL);
+JsonStringStorage JsonStringStorage::_invalid("");
```

Comment:
NULL is macro'd to 0 which is int so throws a build error. You can try
nullptr and see if it builds but I think I tried it.
On Jul 26, 2015 10:41 AM, "Skurydin Alexey" <notifications@github.com>
wrote:

> In Sming/Services/ArduinoJson/src/JsonBuffer.cpp
> <https://github.com/anakod/Sming/pull/182#discussion_r35494126>:
>
> > @@ -14,7 +14,7 @@
> >  using namespace ArduinoJson;
> >  using namespace ArduinoJson::Internals;
> >
> > -JsonStringStorage JsonStringStorage::_invalid(NULL);
> > +JsonStringStorage JsonStringStorage::_invalid("");
>
> Why not NULL or nullptr for invalid values?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/anakod/Sming/pull/182/files#r35494126>.
>